code/src/components/AddTask/index.tsx

```diff
@@ -0,0 +1,54 @@
+import React, { useState } from "react";
+import { useDispatch } from "react-redux";
+import { v4 as uuidv4 } from "uuid";
+import { FaPlus } from "react-icons/fa";
+
+import { Task } from "../../types";
+import tasks from "../../redux/tasksSlice";
+
+import { AddTaskForm, AddTaskInput, AddTaskButton } from "./style";
```

Comment:
remember to organize your imports in a column when you have more than 2 imports from the same library